vignettes/articles/hubEnsembles.Rmd

```diff
@@ -281,19 +291,55 @@ weighted_linear_pool_norm <- hubExamples::forecast_outputs |>
 head(weighted_linear_pool_norm, 10)
 ```
 
-We can also change the distribution that `distfromq` uses to estimate the component models' distributional tails to either log normal or Cauchy using the `tail_dist` argument. (See documentation in the `distfromq` package for more details and function options.) Note, though, that the choice of tail distribution usually does not have a large impact on the resulting ensemble distribution, except in its outer edges.
+## Changing the parametric family used for extrapolation into distribution tails
+
+We can also change the distribution that `distfromq` uses to approximate the tails of component models' predictive distributions to either log normal or Cauchy using the `tail_dist` argument. This choice usually does not have a large impact on the resulting ensemble distribution, though, and can only be seen in its outer edges. (For more details and other function options, see the documentation in the [`distfromq` package](https://reichlab.io/distfromq/).)
 
 ```{r}
 linear_pool_lnorm <- hubExamples::forecast_outputs |>
-  dplyr::filter(!output_type %in% c("median", "sample")) |>
+  dplyr::filter(output_type == "quantile") |>
   hubEnsembles::linear_pool(
     model_id = "linear-pool-lognormal",
     tail_dist = "lnorm"
   )
 linear_pool_cauchy <- hubExamples::forecast_outputs |>
-  dplyr::filter(!output_type %in% c("median", "sample")) |>
+  dplyr::filter(output_type == "quantile") |>
   hubEnsembles::linear_pool(
     model_id = "linear-pool-cauchy",
     tail_dist = "cauchy"
   )
 ```
+
+## Ensembling samples from joint distributions
+
+All output types will summarize predictions from marginal distributions. The sample output type is unique in that **it can additionally represent predictions from joint predictive distributions**. Hence, the sample representation may capture dependence across combinations of multiple modeled task ID variables when recording samples from a joint predictive distribution. In this case, samples with the same index (specified by the `output_type_id`) may be assumed to correspond to a single sample from a joint distribution across multiple combinations of values for the task ID variables.
```

Comment:
```suggestion
All output types will summarize predictions from marginal distributions. The sample output type is unique in that **it can additionally represent predictions from joint predictive distributions**. Hence, the sample representation may capture dependence across multiple combinations of values for modeled task ID variables when recording samples from a joint predictive distribution. In this case, samples with the same index (specified by the `output_type_id`) may be assumed to correspond to a single sample from a joint distribution across multiple combinations of values for the task ID variables.
```